### PR TITLE
Update kops docs

### PR DESCRIPTION
**Pull request by TerryHowe**
Document changes for new kOps scripts

Docs in brach: https://github.com/aws/eks-distro/blob/e2e-docs/development/kops/README.md

### CODE REVIEW

File: development/kops/README.md [hunk 1]
```diff
@@ -34,13 +34,13 @@ EKS-D cluster:
 ```
 
 ### 4. Wait for the Cluster
-It may take a while for the cluster. You can use this script to wait for
-the cluster and apply the AWS IAM authenticator when it comes up.
+It may take a while for the cluster. You can use the `cluster_waith.sh`
```
Comment (micahhausler) [inline development/kops/README.md:37]:
```suggestion
It may take a while for the cluster. You can use the `cluster_wait.sh`
```

**Review by micahhausler — CHANGES_REQUESTED** [review]
One nit

File: development/kops/README.md (hunk 1 above)
Comment (TerryHowe) [reply development/kops/README.md]: Fixed, thanks!

**Review by abhay-krishna — APPROVED** [review]

**Pull request merged**